### ISSUE TITLE
How to correct the problem? 

### ISSUE THREAD

**Issue by tangfayuan** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#142]
UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule.
